### DISCUSSION TITLE
how to access app.mydomain.tld with docker server givne as and run the web client as

### DISCUSSION THREAD

**Discussion by johnsmithodom** [shlinkio/discussions#24] [Help wanted]
after we turn the web client as:
docker run \
    --name shlink-web-client \
    -p 8000:80 \
    -e SHLINK_SERVER_URL=http://www.mydomain.tld:8080 \
    -e SHLINK_SERVER_API_KEY=c6faa02f-2fb7-xxxx-a953-41d01dde1477 \
    shlinkio/shlink-web-client
  
or 
  docker run \
    --name shlink-web-client \
    -p 8000:80 \
    -e SHLINK_SERVER_URL=http://www.mydomain.tld \
    -e SHLINK_SERVER_API_KEY=c6faa02f-2fb7-xxxx-a953-41d01dde1477 \
    shlinkio/shlink-web-client
  
**_Do we need need add the port of the server like how we can access it in the browser like adding port 8080 ?  since trying to access through app.mydomain.tld is not working ??**

root@vps:~# docker ps
CONTAINER ID   IMAGE                    COMMAND                  CREATED          STATUS          PORTS                                       NAMES
7c7a2cb6fd61   shlinkio/shlink:stable   "/bin/sh ./docker-en…"   31 minutes ago   Up 31 minutes   0.0.0.0:8080->8080/tcp, :::8080->8080/tcp   my_shlink
root@vps:~# docker exec -it my_shlink shlink api-key:generate 

 [OK] Generated API key: "c6faa02f-2fb7-xxxx-a953-41d01dde1477"

                    
`
CONTAINER ID   IMAGE                        COMMAND                  CREATED          STATUS          PORTS                                       NAMES
352c3826ed7c   shlinkio/shlink-web-client   "/docker-entrypoint.…"   15 minutes ago   Up 15 minutes   0.0.0.0:8000->80/tcp, :::8000->80/tcp       shlink-web-client
7c7a2cb6fd61   shlinkio/shlink:stable       "/bin/sh ./docker-en…"   2 hours ago      Up 2 hours      0.0.0.0:8080->8080/tcp, :::8080->8080/tcp   my_shlink

Thank you for any tip in advance.

**Comment by Lockszmith-GH** [comment]
Does [this](https://github.com/shlinkio/shlink-web-client/discussions/793) help?

If in my linked example the routes are example.com and shlink.example.com, in your case it would be:
* `example.com` = `www.mydomain.tld:8080`
* `shlink.example.com` = `www.mydomain.tld`

**Reply by Lockszmith-GH** [reply]
note that in both cases, the `SHLINK_SERVER_URL` environment variable needs to be the same, in your case it's the :8080 one.
